snippets/base/forms.py

```diff
@@ -148,6 +149,32 @@ class BaseSnippetAdminForm(forms.ModelForm):
     pass
 
 
+class SnippetChangeListForm(forms.ModelForm):
+    class Meta:
+        model = Snippet
+        fields = ('body',)
```

Comment:
Nit: blank line after the internal class

Reply:
/me wonders why flake8 doesn't complain about this?